lib/parser.js

```diff
@@ -159,13 +159,21 @@ async function customDocumentOperations(js, asyncapiYAMLorJSON, initialFormat, o
     const convert = OPERATIONS.map(async (opName) => {
       const op = channel[opName];
       if (op) {
+        let messages = [];
+        const msg = op.message;
+        if (msg) {
+          if (msg.oneOf) {
```

Comment:
Should we just leave the other options (`allOf` and `anyOf`) for later issues? And are there even use-cases for them? 🤔

Reply:
For messages in Operation Type, AsyncAPI have two only two options: single message or `oneOf` messages. Pic from documentation 

![image](https://user-images.githubusercontent.com/20404945/86720276-27aa0c00-c025-11ea-9c6e-f85f6f82354a.png)

`allOf` and `anyOf` is in Schema Type :)